lib/situation_room/check/request_client.ex

```diff
@@ -0,0 +1,8 @@
+defmodule SituationRoom.RequestClient do
+  @moduledoc """
+  This is a client setup for Tesla - using Hackney instead of default Httpc for ssl
+  """
+  use Tesla
+
+  adapter(Tesla.Adapter.Hackney)
```

Comment:
I'd be curious to try the Finch adapter here instead of Hackney.  Unless there was specific reason that Hackney was chosen?

Reply:
No reason Hackney was chosen - I can certainly use Finch. What kind of perks does Finch have over Hackney?

Reply:
I am currently using Finch and I must say - I did like the error messages that Hackney would give me when I passed invalid URLs into requests. This should not be that much of a problem because in order for a site to be checked, it has to pass an ecto validation when the site is created. Since this is the case, is there any value in testing for invalid URLs when performing site checks?